tests/foreman/ui/test_eol_banner.py

```diff
@@ -0,0 +1,105 @@
+"""Test module for Dashboard UI
+
+:Requirement: Dashboard
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: Dashboard
+
+:Team: Endeavour
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+
```

Comment:
make-docs check fails because of these tokens, could you remove it?
```
* Unexpected tokens:
  Caselevel: Acceptance
  Testtype: Functional
  Upstream: No
```

Reply:
@Gauravtalreja1 thank you for the notice, I removed those